src/benchpress/evaluation/engine.py

```
@@ -52,14 +63,82 @@ def __init__(
             console: Rich console for output formatting (optional)
             streaming: Whether to use streaming API for model generation (optional)
             max_tokens: Maximum number of tokens to generate (optional)
+            sequential: Whether to process examples sequentially (optional, defaults to False)
+            max_concurrency: Maximum number of concurrent requests when running in parallel (optional, defaults to 5)
         """
         self.model = model
-        self.output_dir = Path(output_dir) if output_dir else None
+        self.output_dir = Path(output_dir) if output_dir else (Path.cwd() / 'results').as_posix()
```

Ruff (0.8.2)

Line too long (97 > 88)

(E501)